tst/testbugfix/00005.tst

```diff
@@ -15,4 +15,4 @@ gap> G:=GroupByGenerators( [ [ [ 0, -1, 0, 0 ], [ 1, 0, 0, 0 ],
 Group([ [ [ 0, -1, 0, 0 ], [ 1, 0, 0, 0 ], [ 0, 0, 0, -1 ], [ 0, 0, 1, 0 ] ] 
  ])
 gap> Centralizer(N,G);
-<matrix group of size infinity with 6 generators>
+<matrix group with 6 generators>
```

Comment:
Interesting. Do you by chance know why we don't know that this centralize is infinite anymore?